src/components/elements/input/TextInput.tsx

```diff
@@ -62,7 +61,7 @@ const TextInput = ({
     <TextField
       id={htmlId}
       fullWidth={fullWidth}
-      label={hiddenLabel || horizontal ? undefined : label}
+      label={horizontal ? undefined : label} // hide the label if this text input is part of a horizontal group (informally deprecated)
```

Comment:
https://github.com/open-path/Green-River/issues/7005#issuecomment-2569498578

Based on my search, the `hiddenLabel` prop is not used anywhere, so I'm proposing to remove it entirely, in order to simplify the logic around `ariaLabel`. 

The problem described in the [linked comment](https://github.com/open-path/Green-River/issues/7005#issuecomment-2569498578) arose because `String(label)` returns the string `"undefined"` when label is not passed. 

Previously, we only applied `String(label)` with the `hiddenLabel` prop, but I updated that logic in my [previous PR](https://github.com/greenriver/hmis-frontend/pull/1002#issue-2748013606). (See also historical https://github.com/greenriver/hmis-frontend/pull/675/files#r1512783988)

Reply:
👍